rotkehlchen/tests/unit/test_chains_aggregator.py

```diff
@@ -26,42 +27,81 @@ def test_module_deactivation(blockchain):
         assert module_name not in blockchain.eth_modules
 
 
-def test_filter_active_evm_addresses(blockchain):
+@pytest.mark.parametrize('ethereum_accounts', [[]])
+def test_detect_evm_accounts(blockchain: 'ChainsAggregator'):
+    eth_addy_contract = make_evm_address()
```

Comment:
Can you add a docstring explaining what is getting tested?

```suggestion
def test_detect_evm_accounts(blockchain: 'ChainsAggregator') -> None:
    eth_addy_contract = make_evm_address()
```